### DISCUSSION TITLE
How to resize window

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#452] [Q&A] [answered]
The CNE window is too damn small
How do I resize it to 1440x810 on launch

**Comment by ghost** [comment]
use FlxG.resizeGame

**Reply by THEkatinamicrowave** [reply]
where do I put this?

**Reply by r6915ee** [reply]
You could put it in a Global script and use the method when loading the mod.

**Reply by THEkatinamicrowave** [reply]
which callback is for modLoaded

**Reply by r6915ee** [reply, marked as answer]
My best guess would be to do it something like this:

```haxe
function preStateSwitch() {
    FlxG.resizeWindow(1440, 810);
    FlxG.resizeGame(1440, 810); // comment or remove this line if you don't need to resize the game itself
}
```

Since this script likely won't be in an addons folder, I did not include a mod folder name check, but you may use one if you're putting this in an addon (just import `funkin.backend.assets.ModsFolder` and include an if statement for whether or not `ModsFolder.currentModFolder` is equivalent to your mod's name). Here, the `preStateSwitch` callback is used, as I don't know if there is a callback for loading mods specifically and in case the player might've resized the window on their own by accident. Each time the callback is executed, then the window and game are resized accordingly.
